addons/test_mail/static/tests/activity_tests.js

```diff
@@ -623,6 +623,47 @@ QUnit.module("test_mail", {}, function () {
         }
     );
 
+    QUnit.test("activity view: Domain should not reset on load", async function (assert) {
+            Object.assign(serverData.views, {
+                "mail.test.activity,false,list":
+                    '<tree string="MailTestActivity"><field name="name"/></tree>',
+            });
+            const { env, openView } = await start({
+                mockRPC(route, args) {
+                    if (args.method === "name_search") {
+                        args.kwargs.name = "MailTestActivity";
```

Comment:
Do you need to edit args.kwargs.name ? It s not possible do to do this research with the interface and receive the good value in args.kwargs.name?

Reply:
No I guess, Bad copy paste :cry: